server/routes/custom_panels/panels_router.ts

```diff
@@ -290,11 +321,11 @@ export function PanelsRouter(router: IRouter) {
         );
         return response.ok({
           body: {
-            message: 'Panel PPL Filter Changed',
+            message: "Panel PPL Filter Changed",
           },
         });
       } catch (error) {
-        console.error('Issue in adding query filter', error);
+        console.error("Issue in adding query filter", error);
```

Comment:
curious what formatter are you using? not an issue but dashboards config should make them single quotes
 https://github.com/opensearch-project/OpenSearch-Dashboards/blob/ff23cc0ead19da95cb8e51cb7b2e364bda884a55/.prettierrc#L2